src/Header/Header.css

```diff
@@ -0,0 +1,6 @@
+.headerContainer {
+    background-color: #6C757D;
+    color: #FFFFFF;
+    position: relative;
+    height: 10vh;
```

Comment:
as discussed this should probably not have a set height and rather just fit its contents and a bit of padding